src/environments/environment.dev.ts

```diff
@@ -0,0 +1,7 @@
+import { version } from '../../package.json';
+
+export const environment = {
+    vectorApi: 'https://vector-api.dev.swx-trec.com',
+    production: true,
```

Comment:
```suggestion
    production: false,
```
Does this really matter?

Reply:
I forget, but I think your version is correct. We use a production build, but set production false for dev so we can distinguish between deployed dev and deployed prod…